components/concept/fetch/src/test/java/mozilla/components/concept/fetch/ResponseTest.kt

```diff
@@ -210,7 +211,7 @@ class ResponseTest {
 
         body.string()
 
-        verify(stream).close()
+        verify(stream, atLeastOnce()).close()
```

Comment:
I guess this comes from the double usage of `use`? I guess we could get rid of the `useStream()` block now? 🤔

Reply:
Yes. In my head, I knew we only needed one but it irrationally felt dangerous to write. 😅 We have the test to cover that case though so it should be safe.

> I guess we could get rid of the useStream() block now?

Ah, I see. With these two changes in mind, I could change it to:
```kotlin
        fun string(charset: Charset? = null): String = use {
            // <that big comment>
            stream.reader(charset ?: this.charset).readText()
        }
```

I'll update the code to do this and remove the changes from the test.

Reply:
@pocmo Can you confirm this is the change you were expecting and, if so, flag this to land?

Reply:
Looks good 👍